tests/testthat/_snaps/check_tbl_spl_compound_tid.md

```diff
@@ -0,0 +1,40 @@
+# check_tbl_spl_compound_tid works
+
+    Code
+      check_tbl_spl_compound_tid(tbl, round_id, file_path, hub_path)
+    Output
+      <message/check_success>
+      Message:
+      Task ID combinations across compound idx samples consistent.
+
+---
+
+    Code
+      check_tbl_spl_compound_tid(tbl_error, round_id, file_path, hub_path)
+    Output
+      <warning/check_failure>
+      Warning:
+      Task ID combinations across compound idx samples not consistent.  Sample "1" does not match most prevalent task ID combination for its compound idx. See `errors` attribute for details.
```

Comment:
Totally agree! I don't think it will make it into this release but I've actually got a related issue open on this: https://github.com/Infectious-Disease-Modeling-Hubs/hubValidations/issues/76

Reply:
Perfect, sorry I miss the already open issue